src/components/intro/index.tsx

```diff
@@ -1,26 +1,134 @@
 import { h, Component } from 'preact';
+
+import { bind, linkRef, Fileish } from '../../lib/util';
+import '../custom-els/LoadingSpinner';
+
+import logo from './imgs/logo.svg';
```

Comment:
Is this a preference, or just to get the assets into Webpack?  If the latter we could add all of `/imgs/` via CopyWebpackPlugin.